### PR TITLE
Increases the gap between the rows of icons

### LINKED ISSUE #654: No gap between the icons in hamburger menu
**Description**
There is no gap between the icons in hamburger menu.

**screenshot**
Screenshot 2020-01-19 01:25:01

### PR DESCRIPTION
**Pull request by mohitm15**
Fixes #654 

**Screenshot**

Screenshot 2020-01-19 01:24:28

### CODE REVIEW

**Comment by mohitm15** [comment]
@mariobehling review this

**Comment by prateekiiest** [comment]
@mariobehling would it be fine, if I am reviewing this PR since I am actually not mentoring this year.
I would still be nevertheless happy to do so.

**Review by Chaitanyassr — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -185,6 +185,7 @@
 								<li><a href="https://youtube.com/fossasiaorg" target="_self"><i class="icon social_youtube icon-large"></i></a></li>
 								<li><a href="https://www.flickr.com/photos/fossasia/" target="_self"><i class="icon social_flickr"></i></a></li>
 								<li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li>
+								<br><br><br>
```
Comment (abishekvashok) [inline index.html:188]: Not a robust solution when in desktop mode

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by prateekiiest** [comment]
Is this not resolved yet?

**Pull request closed without merging**